lib/capsize/application.rb

```diff
@@ -0,0 +1,31 @@
+module Capsize
+  module Application
+    def self.load_tasks
+      Application.load_tasks_in_isolation do
+        require 'capistrano/all'
+        cap = Capistrano::Application.new
+        Rake.application = cap
+        cap.init
+        cap.load_rakefile
+        cap.tasks
+      end
+    end
+
+    def self.load_tasks_in_isolation
+      read, write = IO.pipe
+
+      pid = fork do
+        read.close
+        result = yield
+        write.puts result
+        exit!(0)
+      end
+
+      write.close
+      cap_tasks = []
+      read.each_line { |task| cap_tasks << task }
```

Comment:
What this method does is fork and then pipe some output from the parent to the child. It doesn't have anything to do with tasks: your code that actually is task-specific is up on line 10, in an entirely different method. So this method's name and the local variable names should be updated to reflect what is actually going on within this method, not how it happens to be being used in one particular case.